### ISSUE TITLE
utils/indicators.py

### ISSUE THREAD

**Issue by temur3559** [temur3559/UltimateCryptoBot#7]
import numpy as np
import pandas as pd

def compute_rsi(data, window=14):
    delta = pd.Series(data).diff()
    gain = delta.where(delta > 0, 0)
    loss = -delta.where(delta < 0, 0)

    avg_gain = gain.rolling(window=window, min_periods=1).mean()
    avg_loss = loss.rolling(window=window, min_periods=1).mean()

    rs = avg_gain / (avg_loss + 1e-10)
    rsi = 100 - (100 / (1 + rs))
    return rsi

def compute_ema(data, window):
    return pd.Series(data).ewm(span=window, adjust=False).mean()

def compute_macd(data, fast=12, slow=26, signal=9):
    ema_fast = compute_ema(data, fast)
    ema_slow = compute_ema(data, slow)
    macd = ema_fast - ema_slow
    signal_line = compute_ema(macd, signal)
    return macd, signal_line
